.gitignore

```
@@ -10,3 +10,4 @@ docs
 vendor
 node_modules
 .php-cs-fixer.cache
+/runtime/php-cs-fixer.cache
```

We can ignore all the runtime dir
```suggestion
/runtime
```